console/mgmt.go

```diff
@@ -36,10 +37,21 @@ func newMgmt(server *Server) *mgmt {
 }
 
```

Comment:
will need to replace this with the websocket version in the near future. Not a concern for this PR, though